test/WebSocketsTestApp/runtimeconfig.template.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
It's in RC4 CLI 004771 build...not sure which VS build number has matching targets.